web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -220,6 +223,8 @@ function BaseCard({
   );
 }
 
+const infoCardCollapsedAtom = atom<boolean | null>(true);
```

Comment:
Having a shared state for these brings the benefit of reducing Content Layout Shifts when navigating between different zones or toggling the aggregations.